### PR TITLE
Update exercise files

### PR DESCRIPTION
**Pull request by Shuunii1001**
pull_request

### CODE REVIEW

File: Shuunii/module1/user.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Shuunii, Shiyi, Wang
```
Comment (racousin) [inline Shuunii/module1/user.txt:1]: You should remove spaces
```suggestion
Shuunii,Shiyi,Wang
```

**Review by racousin — CHANGES_REQUESTED** [review]

**Review by racousin — CHANGES_REQUESTED** [review]
file name should be user. Not user.txt

**Review by racousin — CHANGES_REQUESTED** [review]
your usernam is Shuunii1001 not Shuunii.

Update the folder-file Shuunii/module1/user.txt to Shuunii1001/module1/user

Update in the file your username from Shuunii,Shiyi,Wang to Shuunii1001,Shiyi,Wang

Commit and push your chnages

**Comment by Shuunii1001** [comment]
Dear sir,
I don’t know how to change the name of folder-file and I can’t gitpush after I edited the file.
Best regards,
Wang Shiyi

---- Replied Message ----
| From | Raphael ***@***.***> |
| Date | 10/07/2024 14:58 |
| To | racousin/data_science_practice_2024 ***@***.***> |
| Cc | Shuunii1001 ***@***.***>,
Author ***@***.***> |
| Subject | Re: [racousin/data_science_practice_2024] Update exercise files (PR #18) |

@racousin requested changes on this pull request.

your usernam is Shuunii1001 not Shuunii.

Update the folder-file Shuunii/module1/user.txt to Shuunii1001/module1/user

Update in the file your username from Shuunii,Shiyi,Wang to Shuunii1001,Shiyi,Wang

Commit and push your chnages

**Comment by Shuunii1001** [comment]
pull_request

**Pull request closed without merging**